mysqld_exporter.go

```diff
@@ -53,6 +53,10 @@ var (
 		"config.my-cnf",
 		"Path to .my.cnf file to read MySQL credentials from.",
 	).Default(path.Join(os.Getenv("HOME"), ".my.cnf")).String()
+	tlsSkipVerify = kingpin.Flag(
+		"tls.skip-verify",
+		"Ignore hostname verification when using a tls connection.",
```

Comment:
This does more than ignore the hostname verification. It ignores all TLS errors.